setup.py

```diff
@@ -17,7 +17,7 @@
     "kanboard": ["kanboard"],
     "keyring": ["keyring"],
     "phabricator": ["phabricator"],
-    "test": ["flake8", "pytest", "responses", "sphinx", "sphinx-click"],
+    "test": ["flake8", "pytest", "responses", "sphinx", "sphinx-click", "docutils"],
```

Comment:
Interesting. I guess we didn't notice that docutils is a direct dependency because its a transitive dependency of several other packages.

Reply:
Yeah, I think I noticed this before I noticed the `-e .[all]`